LibraBFT/Impl/Properties/VotesOnceDirect.agda

```diff
@@ -104,65 +104,66 @@ module LibraBFT.Impl.Properties.VotesOnceDirect where
   -- will involve much of the same complexity, and perhaps be more difficult overall.  Maybe
   -- something like the following could be proved inductively, and then used together with
   -- injectivity properties as mentioned above to determine that the two pids are the same.
+  peerCanSignEp≡ : ∀ {pid v v' pk s'}
+                   → PeerCanSignForPK s' v pid pk
+                   → v ^∙ vEpoch ≡ v' ^∙ vEpoch
+                   → PeerCanSignForPK s' v' pid pk
+  peerCanSignEp≡ (mkPCS4PK 𝓔₁ 𝓔id≡₁ 𝓔inSys₁ mbr₁ nid≡₁ pk≡₁) refl
+    = (mkPCS4PK 𝓔₁ 𝓔id≡₁ 𝓔inSys₁ mbr₁ nid≡₁ pk≡₁)
 
-  postulate
-   MsgWithSig⇒ValidSenderInitialised :
-     ∀ {st pk v}
+  MsgWithSig⇒ValidSenderInitialised :
```

Comment:
Yes! I proved it and I think it can be necessary later to prove `peerCanSign-Msb4`, so let's leave it for now. Ultimately we don't need the `msg∈pool⇒initd` because I can split the step in `oldVoteRound≤lvr` and just use `MsgWithSig⇒ValidSenderInitialised` when the step is init.

Reply:
Ugh, sorry, I didn't read the description well enough or didn't remember when I was reviewing that you had explained about this.